src/widgets/window.js

```diff
@@ -106,7 +104,7 @@ var FlatsealWindow = GObject.registerClass({
 
         this._layoutNotifyId = 0;
         this._updateControlsPlacement();
-        this._headerLeaflet.connect('notify::folded', this._updateControlsPlacement.bind(this));
+        this._contentLeaflet.connect('notify::folded', this._updateControlsPlacement.bind(this));
```

Comment:
Question:

Is there a more "handistic" way to update the controls buttons placement? I would love to get rid of  `_updateControlsPlacement`

Reply:
HdyHeaderGroup